CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Changelog
+## 0.3.3 2018/4/12
```

Comment:
Minor thing, but to keep it aligned with the other entries a `-` between the version and the date would be nice

Reply:
done